.docker/inside-runtests

```diff
@@ -1,10 +1,11 @@
 #!/bin/bash
 set -eu
 service elasticsearch start
+printf "Waiting for elasticsearch to be up and running..."
 for i in {1..4}; do  # wait until elasticsearch is ready
-    curl 'http://localhost:9200/' -s && break
+    curl 'http://localhost:9200/' -s > /dev/null && break
     sleep $i
 done
-curl "http://localhost:9200/_cluster/health/index_name?wait_for_status=green&timeout=20s"
+curl "http://localhost:9200/_cluster/health/index_name?wait_for_status=green&timeout=20s" -s > /dev/null
 cd "${LIBREANT_INST_DIR}"
-./ve/bin/python setup.py test || echo "FAIL"
+./ve/bin/python setup.py test 
```

Comment:
I don't think that this will work. Can the tests ever fail if you do this? I don't think so, because docker doesn't "forward" the exit code of the command it runs inside the container. So the fact that `python setup.py test` returns `1` doesn't make the command `docker run... /libreant/.docker/inside-runlibreant` return 1. Which means that now you have no way of checking if the tests failed. Probably documenting it inside the script would have been better than commenting your change :)

Reply:
@Boyska wrote:
> I don't think that this will work. Can the tests ever fail if you do this? I don't think so, because docker doesn't "forward" the exit code of the command it runs inside the container. So the fact that python setup.py test returns 1 doesn't make the command docker run... /libreant/.docker/inside-runlibreant return 1. Which means that now you have no way of checking if the tests failed. Probably documenting it inside the script would have been better than commenting your change :)

You can easily convince yourself with:
```
docker run busybox /bin/sh -c 'exit 3'
```
I've also tried with a fake failing test and `.docker/test_all.sh` correctly exits with the corresponding nose-test exit status.